hist/hist/inc/TH3.h

```diff
@@ -28,6 +28,14 @@
 class TH2D;
 class TProfile2D;
 
+namespace ROOT::Internal {
+/// Entrypoint for thread-safe filling from RDataFrame.
+template<typename T, typename... Args>
+auto FillThreadSafe(T & histo, Args ... args) -> decltype(histo.FillThreadSafe(args...), void()) {
+   histo.FillThreadSafe(args...);
+}
```

Comment:
Probably nitpick since we're passing integer values anyway
```suggestion
auto FillThreadSafe(T & histo, Args &&... args) -> decltype(histo.FillThreadSafe(args...), void()) {
   histo.FillThreadSafe(std::forward<Args>(args)...);
}
```